### PR TITLE
data:Yasshu55

### PR DESCRIPTION
**Pull request by Yasshu55**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4735"></a>

### CODE REVIEW

File: data/Yasshu55.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/Yasshu55"
+		},
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/eddiejaoude"
+		},
+		{
+			"icon": "FaYoutube",
+			"url": "https://youtube.com/eddiejaoude"
```
Comment (krshkun) [inline data/Yasshu55.json:19]:
```suggestion
```

File: data/Yasshu55.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	],
+
+	"links": [{
+			"group": "Primary Socials",
+			"name": "Twitter: Follow me",
+			"url": "https://twitter.com/eddiejaoude",
+			"icon": "FaTwitter",
+			"color": "#00ACEE"
+		},
+		{
```
Comment (krshkun) [inline data/Yasshu55.json:30]:
```suggestion
```

**Review by krshkun — CHANGES_REQUESTED** [review]
Either replace or remove eddie's links

File: data/Yasshu55.json (hunk 2 above)
Comment (Yasshu55) [reply data/Yasshu55.json]: @krshkun sorry about that, I've made the changes can you please check?

**Review by krshkun — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thank you for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**